openpype/hosts/nuke/api/pipeline.py

```diff
@@ -218,7 +219,9 @@ def _install_menu():
     )
     Context.context_label = label
     context_action = menu.addCommand(label)
-    context_action.setEnabled(False)
+
+    if not ASSIST:
```

Comment:
The context action probably should not be added at all when in assist.
```
if not ASSIST:
    context_action = menu.addCommand(label)
    context_action.setEnabled(False)
    menu.addSeparator()
```

Reply:
I believe only issue is that we are not able to disable the menu item. Not to create one.